src/components/WidgetDrawFeatures/WidgetDrawFeatures.js

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import { VISUALIZE_DRAW_MODES } from './../../constants';
+import Divider from'@material-ui/core/Divider';
+import Button from'@material-ui/core/Button';
+import InputLabel from '@material-ui/core/InputLabel';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+import { GeoJSON } from 'ol/format';
+
+const styles = theme => ({
+  root: {
+
+  }
+});
+
+class WidgetDrawFeatures extends React.Component {
+  static propTypes = {
+    layerCustomFeature: PropTypes.object.isRequired,
+    layerCustomFeatureActions: PropTypes.object.isRequired
+  };
+
+  constructor (props) {
+    super(props);
+    this.state = {
+      name: "",
+      description: ""
+    };
+  }
+
+  componentWillReceiveProps (nextProps) {
+    const { currentSelectedDrawnFeatureProperties } = nextProps.layerCustomFeature;
```

Comment:
Pas certain mais on dirait qu'ici il y a un mismatch entre l'assignation et le check de la ligne suivant. À l'oeil, est-ce que ça ne devrait pas être `!== this.props.layerCustomFeature` considérant que c'est ce qui est attribué à la première ligne?

Reply:
Non